### PR TITLE
Create database using diagrams

### PR DESCRIPTION
**Pull request by saluumaa**
_**The following tasks were completed:**_

- Created a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Added Foreign key indexes to the tables

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name VARCHAR(100),
+  date_of_birth DATE,
+);
+
+CREATE TABLE medical_history (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  admitted_at timestamp,
+  patient_id INT REFERENCES patients(id),
+  status VARCHAR(100),
+);
```
Comment (omarsalem7) [inline schema_based_on_diagram.sql:12]: - [ ] The medical_histories and treatments have many to many relations and you suppose to make a third table to achieve this relation look at the example below:
```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```
- [ ] And consider adding indexes for foreign keys in this table

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  type VARCHAR(100),
+  name VARCHAR(100),
+);
+
+--foreign key Index
+CREATE INDEX indx_invoice_items ON invoice_items (invoice_id);
+CREATE INDEX ON medical_history (patient_id);
+CREATE INDEX ON invoices (medical_history_id);
+CREATE INDEX ON invoice_items (treatment_id);
```
Comment (omarsalem7) [inline schema_based_on_diagram.sql:47]: - [ ] Kindly consider adding the index for `invoice_id` foreign key to improve the performance

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @saluumaa 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights

- The Pull Request has a proper title and description. ✔️
- Table structure created ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (saluumaa) [reply schema_based_on_diagram.sql:47]: Hello @omarsalem7, the Index for invoice_id is already there and it is the first one

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @saluumaa , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the changes requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**